src/main/java/com/moodybluez/enterprise/dao/IMoodDAO.java

```diff
@@ -4,6 +4,12 @@
 
 import java.util.Map;
 
+/**
+ * Allows the user to enter information about their mood
+ *
+ * @param moodId a number that helps identify each mood
+ *
+ */
```

Comment:
for interfaces comment on the specific methods within the interface for example

explained in a different comment